kvbc/include/KVBCInterfaces.h

```diff
@@ -100,11 +100,11 @@ class IReplica {
 
   // Used to read from storage, only when a replica is Idle. Useful for
   // initialization and maintenance.
-  virtual const concord::kvbc::ILocalKeyValueStorageReadOnly& getReadOnlyStorage() = 0;
+  virtual const concord::kvbc::IReader& getReadOnlyStorage() const = 0;
 
   // Used to append blocks to storage, only when a replica is Idle. Useful
   // for initialization and maintenance.
-  virtual Status addBlockToIdleReplica(const concord::kvbc::SetOfKeyValuePairs& updates) = 0;
+  virtual concord::kvbc::BlockId addBlockToIdleReplica(concord::kvbc::categorization::Updates&& updates) = 0;
```

Comment:
Do we even use this method? I don't see how it's any different from regular add block. We should remove it altogether if we can.

Reply:
It is used in Concord and is part of IReplica. It might be possible to remove it.

Reply:
No need to do it in this commit.